### ISSUE TITLE
Aiml inmoov files disclaimer

### ISSUE THREAD

**Issue by moz4r** [MyRobotLab/inmoov#186]
While I'm thinking of it, we need to move disclaimer, before the <aiml> tag inside every files, because programAB engine delete it ( before tag is ok ).
Because sometime it is useful to point licenses type / rights...

**Comment by hairygael** [comment]
Yes the disclaimer is auto produced by programAB, so we could remove it. It
keeps the file a little lighter on Git.
Gael Langevin
Creator of InMoov
InMoov Robot <http://www.inmoov.fr>
@inmoov <http://twitter.com/inmoov>

**Comment by moz4r** [comment]
I think it's important to indicate the license in each file

**Comment by moz4r** [comment]
related : https://github.com/MyRobotLab/myrobotlab/issues/340

**Comment by hairygael** [comment]
Maybe a misunderstanding because of my previous answer.
Yes, we need a disclaimer in each files.
We can use the auto produced disclaimer using ProgramAb for the default
disclaimer and add what is necessary in each file.

Gael Langevin
Creator of InMoov
InMoov Robot <http://www.inmoov.fr>
@inmoov <http://twitter.com/inmoov>

**Comment by kwatters** [comment]
We should make it so ProgramAB does not overwrite the AIML files.  That seems to me to be the underlying issue here.
